### ISSUE TITLE
Getting started: "Current Page" state is not identified for the Subway navigation when the focus is placed on the current page link

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#57]
URL: https://hcdev.openplus.ca/en/documentation/getting-started
User Agent: Chrome, JAWS
Language: English

Issue Description: For the subway navigation, the state "current page" is not identified when focus is placed on the current page link.  "Getting started" link is identified when the focus is placed on the first link. Screen reader users are not aware of that link is for the current page. This issue applied to all the pages that have the subway navigation.
![image](https://user-images.githubusercontent.com/88776404/129026682-5e9c08a9-7ed7-4c76-9095-ec3d4817e13f.png)

Steps to reproduce: Navigate to "Getting started" link, and Inspect the element, aria-current is not used.

Expected results: When focus is placed on the current page link, "Current page" should be included as a part of the link text. Please refer to [Subway navigation: Canada.ca design system](https://design.canada.ca/common-design-patterns/subway-navigation.html) where aria-current is used to identify to screen readers the current page. Or simply remove the hyperlink if it's on the current page.

**Comment by ndolter** [comment]
Validated that the screen reader is now reading 'Getting started current page link'.
